tailwind.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
           "white-100": "#FFFFFF",
           "white-200": "#F1F1F1",
           "gray-100": "#888787",
+          black: "#3C3E64",
```

Comment:
add quotes around black

Reply:
When I run `npm run format` it removes the quotes, but it won't let be commit without running `npm run format`.

Reply:
oh that's weird, i'll take a look at it later but for now it's good